src/routes/tools/wildfire/_constants.js

```diff
@@ -40,19 +40,19 @@ represent a time series of annual/monthly simulations of average Area
 Burned/Decadal Wildfire Probability from individual downscaled GCMs.</p>
 <p>Click on any of the legend keys to highlight corresponding time series.</p>`;
 
+export const DATA_DISCLAIMER = `Locations outside the combined state and federal 
+fire protection responsibility areas were excluded from these wildfire 
+simulations and have no wildfire projections.`;
+
 export const MISSING_DATA_MSG = `This area may contain locations outside the 
 combined fire state and federal protection responsibility areas. 
```

Comment:
update text in `MISSING_DATA_MSG` to "combined state and federal fire protection responsibility areas"

Reply:
This const is actually no longer being used so I'll remove it.